manifests/0000_26_cloud-controller-manager-operator_02_rbac.yaml

```diff
@@ -121,19 +121,19 @@ subjects:
     namespace: openshift-cloud-controller-manager-operator
     name: cluster-cloud-controller-manager
 
-# ---
-# apiVersion: rbac.authorization.k8s.io/v1
-# kind: RoleBinding
-# metadata:
-#   name: cluster-cloud-controller-manager
-#   namespace: openshift-cloud-controller-manager
-#   annotations:
-#     include.release.openshift.io/self-managed-high-availability: "true"
-#     include.release.openshift.io/single-node-developer: "true"
-# roleRef:
-#   kind: Role
-#   name: cluster-cloud-controller-manager
-# subjects:
-#   - kind: ServiceAccount
-#     namespace: openshift-cloud-controller-manager-operator
-#     name: cluster-cloud-controller-manager
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: cluster-cloud-controller-manager
+  namespace: openshift-cloud-controller-manager
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+roleRef:
+  kind: ClusterRole
+  name: admin
```

Comment:
Where is this cluster role defined?

Reply:
https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles One of the default options.